manifests/t/TheStellaTeam/Stella/6.7.1/TheStellaTeam.Stella.locale.en-US.yaml

```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.5.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.5.0.schema.json
+
+PackageIdentifier: TheStellaTeam.Stella
+PackageVersion: 6.7.1
+PackageLocale: en-US
+Publisher: The Stella Team
+PackageName: Stella
+License: GPL-2.0
+Copyright: Copyright © 1995-2024 Bradford W. Mott, Stephen Anthony and The Stella Team
+ShortDescription: Stella Setup

```

Comment:
Please provide a better description of what this application is, does, or is used for